benchmarks/quantities_body.hpp

```diff
@@ -21,50 +21,50 @@ using si::Radian;
 
 namespace {
 
-std::size_t const kDimension = 100;
+std::size_t const dimension = 100;
```

Comment:
This identifier is weird and I have no idea what it means.  Can we find a better one?